protocols/grpc/agent.proto

```diff
@@ -145,7 +145,7 @@ enum IPFamily {
 	v6 = 1;
 }
 
-message IPAddress {
+message IpAddress {
```

Comment:
Why?

I think `IP` should be more common name and follows golint rules.

Reply:
Sure, will do.